handbook/people-ops/spending-company-money.md

```diff
@@ -1,3 +1,67 @@
 # Spending company money
+Please see below for a list of reimbursable expenses (all in USD).
```

Comment:
There are probably some high-level guidelines to add at the top in subsequent bullets (using https://about.gitlab.com/handbook/spending-company-money/ as a guide). A few that seem relevant, and a few additions:

- You are free to purchase these items and expense them without asking permission **as long as it's in the interest of the company**. When in doubt, please contact @sourcegraph/people-ops for clarification before the purchase.
- You likely won't need every single item on this list—please use your best judgment and buy when you need them. Remember, Sourcegraph doesn't have unlimited funds—spend this money as if it was your own.
  - For example, if you intend to [work out of a coworking space](#Ongoing-expenses) most of the time, you may not need a fully equipped home office as well.
- These are average amounts, and should be treated as guidelines, not hard limits. If you expect to significantly exceed the listed amount on any purchase, please notify @sourcegraph/people-ops so we can decide if we need to update this guide.
- If you spend more than $1,000 on any given item, it will be considered property of the company.

... Anything else?

Reply:
>- These are average amounts, and should be treated as guidelines, not hard limits. If you expect to significantly exceed the listed amount on any purchase, please notify @sourcegraph/people-ops so we can decide if we need to update this guide.

My bullet above may actually be incorrect, looking at the numbers below. Were they intended to be maximum amounts, or averages?

Reply:
The majority of the amounts are based on what we currently spend on these items, so I was thinking of them as maximums (unless there needed to be an exception).